applications/CoSimulationApplication/python_scripts/co_simulation_tools.py

```diff
@@ -15,6 +15,13 @@ def cs_print_warning(label, *args):
     KM.Logger.PrintWarning(colors.red("Warning: ")+colors.bold(label), " ".join(map(str,args)))
 
 
+def UsingPyKratos():
+    for i_path in KM.__path__:
+        if "pyKratos" in i_path:
+            return True
+    return False
```

Comment:
@adityaghantasala an alternative would be a global var (set when importing, i.e. from__init__.py) but I think this is the better solution